### PR TITLE
fix(notifications): Notification Settings Toggle

### PR DESCRIPTION
**Pull request by mgaeta**
This PR solves two bugs:
- filter out projects from notification settings where the user doesn't have access
- hides notification setting fine-tuning when all preferences are "never".

Screen Shot 2021-05-18 at 11 04 02 AM

### CODE REVIEW

**Comment by ceorourke** [comment]
Can you add a mailto for feedback? Something like this, I don't think we have a specific design for it: 

Screen Shot 2021-05-18 at 12 19 21 PM

**Comment by ceorourke** [comment]
Would it be too much to ask to sneak in flipping the default text? e.g. where it says "Only Subscribed Issues (default)" flip it to "Default (Only Subscribed Issues)". 

Also, we wanted to have front end tests for this but I'm okay with creating those separately in the name of getting this out on time.

File: static/app/views/settings/account/notifications/notificationSettings.tsx [hunk 1]
```diff
@@ -256,58 +328,92 @@ class NotificationSettings extends AsyncComponent<Props, State> {
...
-    const groupedParents = this.getGroupedParents();
-    const userData = this.getUserDefaultValues();
-    const parentData = this.getParentData();
-    const [formTitle, fields] = this.getDefaultSettings();
 
     return (
       <React.Fragment>
         <SettingsPageHeader title={title} />
         {description && <TextBlock>{description}</TextBlock>}
+        <FeedbackAlert type="info" icon={<IconInfo />}>
```
Comment (ceorourke) [inline static/app/views/settings/account/notifications/notificationSettings.tsx:378]: can you feature flag this?

Comment (mgaeta) [reply static/app/views/settings/account/notifications/notificationSettings.tsx:378]: The entire page is behind the flag.

**Review by ceorourke — APPROVED** [review]

**Pull request merged**